src/index.js

```diff
@@ -72,10 +89,27 @@ class RecPlayer extends Component {
     )(cnt => {
       this.cnt = cnt;
       this.autoResize();
-      urls.recUrl && cnt.loadReplay(urls.recUrl);
+      if (urls.recUrl) {
+        const recs = urls.recUrl.split(';');
+        if (Array.isArray(recs)) {
```

Comment:
Isn't this always an array since using split() method? If ; is not found it's an array with one url?

Reply:
yea you're right :)